zcash_client_backend/src/data_api.rs

```diff
@@ -1,4 +1,63 @@
-//! Interfaces for wallet data persistence & low-level wallet utilities.
+//! Utilities for Zcash wallet construction
+//! =======================================
```

Comment:
`rustdoc` uses Markdown:
```suggestion
//! # Utilities for Zcash wallet construction
```